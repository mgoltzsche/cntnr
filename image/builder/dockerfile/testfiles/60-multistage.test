FROM alpine:3.7
RUN printf '#!/bin/sh\necho hello from copied app\n' > /script.sh && chmod 744 /script.sh

FROM scratch
COPY --from=0 /bin/busybox /bin/
COPY --from=0 /lib/*musl* /lib/
WORKDIR /bin
ENTRYPOINT ["/bin/busybox"]
COPY --from=0 script.sh copiedscript.sh
RUN ["/bin/busybox", "sh", "-c", "[ ! -d /var ] || (ls -la /; false)"]
RUN ["/bin/busybox", "sh", "/bin/copiedscript.sh"]

# ASSERT CFG /config/Env=