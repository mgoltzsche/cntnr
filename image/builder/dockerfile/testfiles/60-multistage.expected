FROM alpine:3.7
NAME 0
RUN "/bin/sh" "-c" "printf '#!/bin/sh\\necho hello from copied app\\n' > /script.sh && chmod 744 /script.sh"

FROM scratch
NAME 1
COPY image="0" "/bin/busybox" "/bin/" nil
COPY image="0" "/lib/*musl*" "/lib/" nil
WORKDIR /bin
ENTRYPOINT "/bin/busybox"
COPY image="0" "script.sh" "copiedscript.sh" nil
RUN "/bin/busybox" "sh" "-c" "[ ! -d /var ] || (ls -la /; false)"
RUN "/bin/busybox" "sh" "/bin/copiedscript.sh"